automation/features/step-definitions/staking.steps.js

```diff
@@ -48,23 +34,228 @@ When(/^I click on use maximum button$/, () => {
     reverse: false,
     timeoutMsg: "callout was not found",
   });
-  if (browser.getByTestId("associate-more-tokens-btn").isDisplayed()) {
-    browser.getByTestId("associate-more-tokens-btn").click();
-  } else {
-    browser.getByTestId("associate-tokens-btn").click();
-  }
+  if (stakingPage.associateMoreTokensBtn.isDisplayed()) {
+    stakingPage.associateMoreTokensBtn.click();
+  } else stakingPage.associateTokensBtn.click();
```

Comment:
Inconsistent synatx:
```suggestion
  if (stakingPage.associateMoreTokensBtn.isDisplayed()) {
    stakingPage.associateMoreTokensBtn.click();
  } else { 
    stakingPage.associateTokensBtn.click();
  }
```

Reply:
There's several places this is the case. Can we fix them all please.